tests/notifier/notifier_test.py

```diff
@@ -17,8 +17,136 @@
 import mock
 
 from google.cloud.forseti.notifier import notifier
+from google.cloud.forseti.notifier.notifiers import email_violations
+from google.cloud.forseti.notifier.notifiers import gcs_violations
 from tests.unittest_utils import ForsetiTestCase
 
+FAKE_NOTIFIER_CONFIGS = {
+    'resources': [
+        {'notifiers': [
+            {'configuration': {
+                'sendgrid_api_key': 'SG.HmvWMOd_QKm',
+                'recipient': 'ab@cloud.cc',
+                'sender': 'cd@ex.com'},
+                 'name': 'email_violations'},
```

Comment:
Please order the elements in the fake config, to match the order in the actual config.  This will make comparison, and future maintenance so much easier.

https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/configs/server/forseti_conf_server.yaml.in#L84-L97

Reply:
Fixed.